src/components/Table/Table.jsx

```diff
@@ -164,6 +165,8 @@ const propTypes = {
       onApplyBatchAction: PropTypes.func,
       /** Apply a search criteria to the table */
       onApplySearch: PropTypes.func,
+      onSaveCurData: PropTypes.func,
```

Comment:
Not a big fan of using `Cur` there are many other props larger than this one, and none of them are using short forms.  Personally i'd just use `saveData` why do we need `Cur` or `Current`.  Is there any other kind of data that is being saved?

Reply:
I agree, I'll change `onSaveCurData ` to `saveData` or `onSaveData`. Do you think `saveData` is better than `onSaveData `? Because I named the prop according to `onApplySearch ` which is also a table toolbar action.

Reply:
I'm ok with 'onSaveData' since 'on' properties are self describing as callbacks/events.

Reply:
got it.